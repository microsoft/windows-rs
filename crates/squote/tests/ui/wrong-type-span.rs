use quote::quote_spanned;

fn main() {
    let span = "";
    let x = 0;
    quote_spanned!(span=> #x);
}
