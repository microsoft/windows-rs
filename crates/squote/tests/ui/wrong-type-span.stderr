error[E0308]: mismatched types
 --> $DIR/wrong-type-span.rs:6:20
  |
6 |     quote_spanned!(span=> #x);
  |     ---------------^^^^-------
  |     |              |
  |     |              expected struct `proc_macro2::Span`, found `&str`
  |     expected due to this
